src/modules/helpers/index.ts

```diff
@@ -797,6 +797,25 @@ export class HelpersModule {
     return object[key];
   }
 
+  /**
+   * Returns a random [key, value] pair from given object or throws an Error if no key could be found.
```

Comment:
```suggestion
   * Returns a random `[key, value]` pair from given object or throws an Error if no key could be found.
```

Reply:
```suggestion
   * Returns a random `[key, value]` pair from the given object.
```

The throwing statement should be explained in the `@throw` tag as [@matthewmayer](https://github.com/faker-js/faker/pull/2123/files#r1183649038) pointed out.